File: Assignments/Assignment_20/p1.c
#include<stdio.h>
void main(){

        int size,index;
        printf("Enter the size of array:\n");
        scanf("%d",&size);
        int arr[size];
        for(int i=0;i<size;i++){

                printf("Enter the arr element %d : ",i+1);
                scanf("%d",&arr[i]);
        }printf("Enter the index to search\n");
        scanf("%d",&index);
        if(index>=size) {
                printf("The index is greater than the size of array\n");
        }else{

                printf("The element at the index %d : %d\n",index,arr[index]);
        }
}

File: Flutter/Assignment/List using Map/p1.dart
import "Dart:io";
void main(){
	print("Enter the size of the list:");
	int? size=int.parse(stdin.readLineSync()!);
	List num=[];
	for(int i=0;i<size;i++){
		int? x=int.parse(stdin.readLineSync()!);
		num.add(x);
	}
	print("Enter an index to search:");
	int? index=int.parse(stdin.readLineSync()!);
	if(index >=0 && index<num.length ){

		print("${num[index]} is present at index $index");
		num.removeAt(index);
		print("Element removed");
	}else{
		print("index out of bound");
	}
}
